src/main/java/games/strategy/engine/data/GameDataMemento.java

```diff
@@ -0,0 +1,136 @@
+package games.strategy.engine.data;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+
+import games.strategy.util.Version;
+import games.strategy.util.memento.MementoExporter;
+import games.strategy.util.memento.MementoImportException;
+import games.strategy.util.memento.MementoImporter;
+import games.strategy.util.memento.PropertyBagMementoExporter;
+import games.strategy.util.memento.PropertyBagMementoImporter;
+
+/**
+ * Provides factory methods for creating objects that can import mementos to and export mementos from game data.
+ */
+public final class GameDataMemento {
+  @VisibleForTesting
+  static final String SCHEMA_ID = "application/x.triplea.game-data-memento";
+
+  /**
+   * An immutable collection of default memento export options.
+   *
+   * <p>
+   * The key is the option name. The value is the option value.
+   * </p>
+   */
+  public static final Map<ExportOptionName, Object> DEFAULT_EXPORT_OPTIONS_BY_NAME = newDefaultExportOptionsByName();
+
+  private GameDataMemento() {}
+
+  @VisibleForTesting
+  interface PropertyNames {
+    String NAME = "name";
+    String VERSION = "version";
+  }
+
+  private static Map<ExportOptionName, Object> newDefaultExportOptionsByName() {
+    return Maps.immutableEnumMap(ImmutableMap.of(
+        ExportOptionName.EXCLUDE_DELEGATES, false));
```

Comment:
Thank you for the previous context. Do you see other options coming down the road? I think in the end we will want one operation to serialize the game data. That simplifies reading it, no special cases. Regardless, if there are no other rules, then we can handle the exclude delegate case more directly perhaps.